client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -557,11 +575,13 @@ export default {
 
     initPagination (data) {
       const dataPag = data.meta.pagination
+      const currentPage = Number(JSON.parse(window.sessionStorage.getItem([this.storageKeyPagination])).currentPage)
```

Comment:
Why are you using sessionStorage (and not localStorage) here? `sessionStorage` [is unique to one browser tab](https://developer.mozilla.org/en-US/docs/Web/API/Window/sessionStorage) and is cleared when this tab is closed. I think we might very well use `localStorage` here.

Reply:
I'm not sure about the reasons for `sessionStorage`. @muellerdemos Do you know more about that?

Reply:
Done in [9af8277](https://github.com/demos-europe/demosplan-core/pull/1342/commits/9af8277525160ae9e7aee3a71b98cac13b5abe87). 
I also updated the pagination storage implementation (I realized we never used the stored values for the API call).